src/background/browserAction.ts

```diff
@@ -20,14 +20,25 @@ import { ensureContentScript } from "@/background/util";
 import { Tabs } from "webextension-polyfill";
 import { rehydrateSidebar } from "@/contentScript/messenger/api";
 import { executeScript, isScriptableUrl } from "webext-content-scripts";
+import webextAlert from "@/background/webextAlert";
+import { isMac } from "@/utils";
 
 // The sidebar is always injected to into the top level frame
 const TOP_LEVEL_FRAME_ID = 0;
 
 async function handleBrowserAction(tab: Tabs.Tab): Promise<void> {
   const url = String(tab.url);
+
+  const extensionConsoleUrl = browser.runtime.getURL("");
```

Comment:
I think the intent is to specifically target the options page since calling `openOptionsPage` gives no feedback if you're already on it. 

```suggestion
  const optionsPage = browser.runtime.getURL("options.html");
```